src/routes/views/overview/components/dashboardWidgetBase.tsx

```diff
@@ -262,12 +263,14 @@ class DashboardWidgetBase extends React.Component<DashboardWidgetProps> {
           formatOptions={trend.formatOptions}
           formatter={chartFormatter || formatUnits}
           height={height}
+          intl={intl}
```

Comment:
We shouldn't pass `intl` as a prop. We've refactored the UI to remove this. Instead, we should wrap the component with `injectIntl`.

That said, I don't believe `ReportSummaryTrend` needs to localize itself, since it just contains a chart.

Both `TrendChart` and `DailyTrendChart` are already wrapped by `injectIntl`, so I'm curious why intl was used here?

Reply:
It's weird... I never made any related changes to the component... but I'm getting the following without intl

> TS2741: Property 'intl' is missing in type '{ adjustContainerHeight: boolean; containerHeight: number; currentData: ChartDatum[]; forecastData: any; forecastConeData: any; formatOptions: FormatOptions; ... 8 more ...; chartName: string; }' but required in type 'WrappedComponentProps<"intl">'.

Reply:
`intl` being required seems to be due to adding a property to `TrendChartOwnProps`, any idea why this would cause this?

Reply:
I modified the code locally and was able to remove the `i18n` prop by making `chartName` optional.

Reply:
Actually it's because `DailyTrendChart` does not support the `chartName` prop. 

`ReportSummaryComponent` is providing `chartName` to either chart `ReportSummaryDailyTrend` (i.e., the `DailyTrendChart`) or `ReportSummaryTrend` (i.e., the `TrendChart`).

So, both charts must support the same required prop or you must make it optional

Reply:
ah... wonder why making it optional didn't work for me yesterday... maybe i just didn't wait long enough for the ide to update.. still seems weird that it would complain about intl rather than chartname.